src/services/userService.ts

```diff
@@ -0,0 +1,40 @@
+import { AxiosRequestConfig } from 'axios';
+
+import { User } from 'pnc-api-types-ts';
+
+import { pncClient } from 'services/pncClient';
+
+/**
+ * Class managing information about user.
+ */
+class UserService {
```

Comment:
we should split it into UserService (API communication, just functions, not class) and UserManager (user state management)